src/components/chart/Chart.vue

```diff
@@ -153,6 +153,9 @@ export default class extends Mixins(PaneMixin) {
         case this.paneId + '/SET_GRIDLINES':
           this.updateGridlines(mutation.payload.type)
           break
+        case this.paneId + '/SET_WATERMARK':
+          this.updateWatermark()

```

Comment:
Better call `this._chartController.updateWatermark()` directly (and remove updateWatermark in vue component)